DOC-2239 delete RERC (#2564)
RedisLabs/redislabs-docs: content/kubernetes/delete_custom_resources.md

  kubectl get reaadb -o=jsonpath='{range .items[*]}{.metadata.name}'
  ```

## Delete a remote cluster (RERC)

{{<note>}}The Redis Enterprise Active-Active database (REAADB) custom resource is currently in public preview. View [Preview REAADB]({{<relref "/kubernetes/active-active/preview/">}}) for more details.{{</note>}}

> 1. Verify the RERC you want to delete isn't listed as a particpating cluster in any REAADB resources.

Author: ```suggestion
1. Verify the RERC you want to delete isn't listed as a participating cluster in any REAADB resources.
```

>   If an RERC is still listed as a participating cluster in any database, the deletion will fail.

Reviewer: The numbered list isn't rendering correctly. I think adding additional spaces to increase the indentation will fix it. This is also an issue for [Delete an Active-Active database](https://docs.redis.com/staging/DOC-2239/kubernetes/delete_custom_resources/#delete-an-active-active-database-reaadb).
```suggestion
    If an RERC is still listed as a participating cluster in any database, the deletion will fail.
```

>     If an RERC is still listed as a participating cluster in any database, the deletion will fail.

Reviewer 2: maybe use this wording:
"""
the deletion will fail
"""
as:
the deletion will be blocked

>     If an RERC is still listed as a participating cluster in any database, the deletion will fail.

Author: ```suggestion
    If an RERC is still listed as a participating cluster in any database, the deletion will be blocked.
```

1. On one of the existing participating clusters, delete the RERC (substituting `<rerc-name>` with your database name).

>   ```sh

Reviewer: Related to the numbered list issue:
```suggestion
    ```sh
```

  kubectl delete rerc <rerc-name>

>   ```

Reviewer: Related to the numbered list issue:
```suggestion
    ```
```

1. Verify the RERC no longer exists.

>   ```sh

Reviewer: Related to the numbered list issue:
```suggestion
    ```sh
```

  kubectl get rerc -o=jsonpath='{range .items[*]}{.metadata.name}'

>   ```

Reviewer: ```suggestion
    ```
```
